ras_backstage/authentication/token_decoder.py

```diff
@@ -0,0 +1,35 @@
+import logging
+
+from flask import abort
+import jwt
+
+from ras_backstage.authentication.uaa import get_uaa_public_key
+
+
+logger = logging.getLogger(__name__)
+
+USER_ID_CLAIM = "user_id"
+
+
+def decode_access_token(access_token, verify=True):
+
+    try:
+        decoded_jwt = jwt.decode(
+            access_token,
+            verify=verify,
+            algorithms=None,
+            key=get_uaa_public_key(),
+            audience='ras_backstage',
+            leeway=10,
+        )
+        return decoded_jwt
+    except jwt.DecodeError:
+        logger.error(f"Unable to decode token {access_token} - confirm the UAA public key is correct")
+        abort(500)
+
+
+def get_user_id(access_token, verify=True):
+    decoded_jwt = decode_access_token(access_token, verify)
+    user_id = decoded_jwt.get(USER_ID_CLAIM)
```

Comment:
Why not just `decoded_jwt.get('user_id')`?

Reply:
Its the java in me, I naturally want to make strings constants